test/build/resolve-imports/src/index.html

```diff
@@ -21,18 +21,36 @@
       console.log(sort, sort_, sort__);
 
       // Importing a directory index.js file
-      import components from './components'; // relative import 
+      import components from './components'; // relative import
       import components_ from './components/index'; // relative import with index appended
       import components__ from './components/index.js'; // relative import with index appended
       import components___ from '@app/components'; // bare import using alias
       import components____ from '@app/components/index'; // bare import using alias and index appended
       import components_____ from '@app/components/index.js'; // bare import using alias and index.js appended
-      console.log(components, components_, components__, components___, components____, components_____);
+      console.log(
+        components,
+        components_,
+        components__,
+        components___,
+        components____,
+        components_____,
+      );
```

Comment:
unrelated question @FredKSchott: did you ever discuss to run `prettier` on other files than just `snowpack/src/**/*.ts`? It would avoid unrelated changes such as this. No big deal, I'm just wondering

Reply:
I’d be down for that. Just a Prettier PR.

Reply:
Do you just mean a one-time pass on all files, or a continuous prettier formatter on all test files, on every PR?

Pro the former, slightly anti the later (mostly just as "not worth the trouble").